src/index.html

```diff
@@ -9,6 +12,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#apple" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#samsung" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#smartphones" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item" data-qa="hover">
+            <a href="#laptopsandcomputers" class="nav-link">Laptops & Computers</a>
```

Comment:
please move the `data-qa="hover"` attribute to `< a/>` tag - it's more explicit like that

Reply:
done in solution3